src/general.rs

```diff
@@ -253,7 +280,7 @@ pub fn initialize(
         root_uri: Some(Url::from_file_path(root_path).unwrap()),
         root_path: None,
         trace: Some(TraceOption::Off),
-        workspace_folders: None,
+        workspace_folders: workspace_folders_from_env(),
```

Comment:
I've hit a similar problem with bash-language-server so I've added shim support
for workspace/worspaceFolders in 2b723953e99ae034d5e595eafd83d5bc8417a9cb.
We currently always send the root path.

We don't really support multiple root paths for now but there might be a use case where it's better to be configurable.
I'd like to to avoid an environment variable (since a Kakoune process might visit several different projects).